### DISCUSSION TITLE
rake aborted! ActiveRecord::RecordInvalid: Validation failed: Main display image Paperclip::Errors::NotIdentifiedByImageMagickError, Origin picture Paperclip::Errors::NotIdentifiedByImageMagickError

### DISCUSSION THREAD

**Discussion by nuel247** [ImageMagick/ImageMagick#6286] [Help]
### ImageMagick version

7.1.1-8 (Beta) 

### Operating system, version and so on

Ubuntu OS (WSL) , ImageMagick 7.1.1-8 (Beta) Q16-HDRI x86_64 21d049b30:20230416, Ruby version 2.7.8

### Description

Hi 

I am tryna get my Dev environment set for a Ruby on rails application. 

I tried passing this command  `bundle exec rake dm:reset_up` and came across this error message 

```
rake aborted!
ActiveRecord::RecordInvalid: Validation failed: Main display image Paperclip::Errors::NotIdentifiedByImageMagickError, Origin picture Paperclip::Errors::NotIdentifiedByImageMagickError
```

I have upgraded my ImageMagick to the 7.1.1-8 beta version but I am still having the error message. Any help will be truly appreciated

**Comment by dlemstra** [comment]
It looks like you should ask your question here instead: https://github.com/thoughtbot/paperclip? This doesn't look like an issue in ImageMagick.
